apps/test/unit/templates/instructions/InlineAudioTest.js

```diff
@@ -3,7 +3,8 @@ import {mount} from 'enzyme';
 import sinon from 'sinon';
 import {setExternalGlobals} from '../../../util/testUtils';
 import React from 'react';
-import {StatelessInlineAudio} from '@cdo/apps/templates/instructions/InlineAudio';
+import {UnconnectedInlineAudio as InlineAudio} from '@cdo/apps/templates/instructions/InlineAudio';
+import {AudioQueue} from '@cdo/apps/templates/instructions/AudioQueue';
```

Comment:
Is there a way we could add a test where multiple audio items are input, and the test ensures they are played sequentially? Or maybe a test that ensures that the AudioQueue component functions properly?